developer-docs-site/static/move_demo/sources/ACLMessageBoard.move

```diff
@@ -0,0 +1,98 @@
+module Examples::Messageboard {
+    use Std::Vector;
+    use Std::Signer;
+    use CoreFramework::Timestamp;
+    use Std::ACL::Self;
+    use Std::Event::{Self, EventHandle};
+
+    // Error map
+    const ACCOUNT_NOT_IN_ACL: u64 = 0;
+    const CANNOT_REMOVE_ADMIN_FROM_ACL: u64 = 1;
+
+
+    struct Messageboard has key, store {
+        participants: ACL::ACL,
+        notice: vector<u8>
+    }
+
+    struct NoticeChangeEventHandle has key, store {
+        change_events: EventHandle<NoticeChangeEvent>
+    }
+    struct NoticeChangeEvent has store, drop {
+        timestamp: u64,
+        sender: address,
+        message: vector<u8>,
+        board: address
+    }
+
+    /// init the message board and move the resource to signer
+    public fun message_board_init_internal(account: &signer){
+        let board = Messageboard {
+            participants: ACL::empty(),
+            notice: Vector::empty<u8>()
+        };
+        ACL::add(&mut board.participants, Signer::address_of(account));
+        move_to(account, board);
+    }
+
+    /// expose the function as a transaction interface
+    public(script) fun message_board_init(account: signer) {
+        message_board_init_internal(&account);
+    }
+
+    public fun view_message(board_addr: address) : vector<u8> acquires Messageboard {
+        let board = borrow_global<Messageboard>(board_addr);
+        board.notice
```

Comment:
does this copy or move the vector?

Reply:
It should be copy. For global storage resource, I think only move_to and move_for can move the value.

Reply:
hmmm, I tried to view message multiple times in test and works fine. But since move doesn't promise implicit copy. will change to *& to do explicit inner copy.

Reply:
*& seems like magic operator, can't we just use `copy board.notice`?